pvlib/test/test_singlediode_methods.py

```diff
@@ -125,3 +126,135 @@ def test_brentq_fs_495():
+
+
+def test_pvsyst_fs_495_recombination_loss():
+    """test pvsystem.singlediode with Brent method on SPR-E20-327"""
+    poa, temp_cell = 1000.0, 25.0  # test conditions
+
+    # first evaluate DeSoto model
+    cec_fs_495 = CECMOD.First_Solar_FS_495  # CEC parameters for
+    il_cec, io_cec, rs_cec, rsh_cec, nnsvt_cec = pvsystem.calcparams_desoto(
+        effective_irradiance=poa, temp_cell=temp_cell,
+        alpha_sc=cec_fs_495.alpha_sc, a_ref=cec_fs_495.a_ref,
+        I_L_ref=cec_fs_495.I_L_ref, I_o_ref=cec_fs_495.I_o_ref,
+        R_sh_ref=cec_fs_495.R_sh_ref, R_s=cec_fs_495.R_s,
+        EgRef=1.475, dEgdT=-0.0003
+    )
+    voc_est_cec = estimate_voc(photocurrent=il_cec, saturation_current=io_cec,
+                               nNsVth=nnsvt_cec)
+    vd_cec = np.linspace(0, voc_est_cec, 1000)
+    desoto = bishop88(
+        diode_voltage=vd_cec, photocurrent=il_cec, saturation_current=io_cec,
+        resistance_series=rs_cec, resistance_shunt=rsh_cec, nNsVth=nnsvt_cec
+    )
+
+    # now evaluate PVSyst model with thin-film recombination loss current
+    pvsyst_fs_495 = PVSYST_FS_495
+    x = pvsystem.calcparams_pvsyst(
+        effective_irradiance=poa, temp_cell=temp_cell,
+        alpha_sc=pvsyst_fs_495['alpha_sc'],
+        gamma_ref=pvsyst_fs_495['gamma_ref'],
+        mu_gamma=pvsyst_fs_495['mu_gamma'], I_L_ref=pvsyst_fs_495['I_L_ref'],
+        I_o_ref=pvsyst_fs_495['I_o_ref'], R_sh_ref=pvsyst_fs_495['R_sh_ref'],
+        R_sh_0=pvsyst_fs_495['R_sh_0'], R_sh_exp=pvsyst_fs_495['R_sh_exp'],
+        R_s=pvsyst_fs_495['R_s'],
+        cells_in_series=pvsyst_fs_495['cells_in_series'],
+        EgRef=pvsyst_fs_495['EgRef']
+    )
+    il_pvsyst, io_pvsyst, rs_pvsyst, rsh_pvsyst, nnsvt_pvsyst = x
+    voc_est_pvsyst = estimate_voc(photocurrent=il_pvsyst,
+                                  saturation_current=io_pvsyst,
+                                  nNsVth=nnsvt_pvsyst)
+    vd_pvsyst = np.linspace(0, voc_est_pvsyst, 1000)
+    pvsyst = bishop88(
+        diode_voltage=vd_pvsyst, photocurrent=il_pvsyst,
+        saturation_current=io_pvsyst, resistance_series=rs_pvsyst,
+        resistance_shunt=rsh_pvsyst, nNsVth=nnsvt_pvsyst,
+        d2mutau=pvsyst_fs_495['d2mutau'],
+        cells_in_series=pvsyst_fs_495['cells_in_series']
+    )
+
+    # test expected change in max power
+    assert np.isclose(max(desoto[2]) - max(pvsyst[2]), 0.01949420697212645)
+    # test expected change in short circuit current
+    isc_desoto = np.interp(0, desoto[1], desoto[0])
+    isc_pvsyst = np.interp(0, pvsyst[1], pvsyst[0])
+    assert np.isclose(isc_desoto - isc_pvsyst, -7.955827628380874e-05)
+    # test expected change in open circuit current
+    voc_desoto = np.interp(0, desoto[0][::-1], desoto[1][::-1])
+    voc_pvsyst = np.interp(0, pvsyst[0][::-1], pvsyst[1][::-1])
+    assert np.isclose(voc_desoto - voc_pvsyst, -0.04184247739671321)
+    return desoto, pvsyst
+
+
+if __name__ == '__main__':
```

Comment:
Although great for development and testing, I believe we don't keep `if __name__ == '__main__':` blocks in modules.

Reply:
✔️ agreed, removed